src/routes/chat/ChatRooms/ChatArchiveButton.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useCallback } from 'react';
+import { reqArchive } from 'reducers/ChatDux';
+import { useDispatch, useSelector } from 'react-redux';
+
+const ChatArchiveButton = ({ isArchived }) => {
+  const dispatch = useDispatch();
+  const chatRoomId = useSelector(state => state.chat.chatRoomId);
+  const userType = useSelector(state => state.misc.userType);
+
+  const archiveCallback = useCallback(() => {
```

Comment:
Same as above, it is not used in `useEffect` why do we need to use `useCallback`